shortly/app/__init__.py

```diff
@@ -1 +1 @@
-from .main import app
+from shortly.app.entrypoints.http.fastapi.main import app
```

Comment:
I would consider adding content to `entrypoint/http/__init__.py`:

```python
from fastapi.main import app
```

And then we could simply use:

```
from shortly.app.entrypoints.http import app
```

I don't think that we have to go so deep into import levels.